rust/examples/pdb-ng/src/symbol_parser.rs

```diff
@@ -2010,13 +2010,13 @@ impl<'a, S: Source<'a> + 'a> PDBParserInstance<'a, S> {
                 self.log(|| format!("Register {:?} ==> {:?}", reg, xreg));
                 self.arch
                     .register_by_name(xreg.to_string().to_lowercase())
-                    .map(|reg| reg.id() as i64)
+                    .map(|reg| reg.id().0 as i64)
```

Comment:
`id().into()`?

Reply:
Maybe, but not using `Into`. Maybe something like `reg.id().into_inner()` or `reg.raw_id()`.

Reply:
The `as i64` can be replaced with `.into()`. But the `.0` can only be replaced with a appropriated named function.